netbox/dcim/views.py

```diff
@@ -2856,7 +2856,10 @@ def get_extra_context(self, request, instance):
         vdc_table.configure(request)
 
         # Get bridge interfaces
-        bridge_interfaces = Interface.objects.restrict(request.user, 'view').filter(bridge=instance)
+        bridge_interfaces = Interface.objects.restrict(request.user, 'view').filter(bridge=instance).select_related(
+            'device', 'module', 'parent', 'bridge', 'lag', 'vrf', 'untagged_vlan', 'qinq_svlan',
+            'primary_mac_address'
+        ).prefetch_related('tagged_vlans', 'wireless_lans', 'vdcs')
```

Comment:
Please remove these `select_related()` and `prefetch_related()` calls, they are handled via `BaseTable.__init__()`.